snorkeling_helper/label_functions/__init__.py

```diff
@@ -1,3 +1 @@
-__all__ = ['DaG']
-from .global_helpers import *
-from .disease_gene import DaG
+__all__ = []
```

Comment:
Does this `__all__` attribute have any function? I never use module-level attributes but I want to learn if there is something interesting.

Reply:
>Does this __all__ attribute have any function? I never use module-level attributes but I want to learn if there is something interesting.

[Detailed explanation here](https://stackoverflow.com/questions/44834/can-someone-explain-all-in-python). Basically if a user were to do ```python from module import *``` `__all__` tells the interpreter which variables to send back to the import statement.

Reply:
Oh that's very nice!